src/modules/campaign-monitor/services/campaign-monitor.service.ts

```diff
@@ -58,54 +66,173 @@ export class CampaignMonitorService {
 			}
 		});
 
-		if (recipients.length === 0) {
-			this.logger.error(
-				`Mail will not be sent - no recipients. emailInfo: ${JSON.stringify(emailInfo)}`
+		const data: CampaignMonitorData = {
+			to: recipients,
+			consentToTrack: 'unchanged',
+			data: this.convertVisitToEmailTemplateData(emailInfo.visit),
+		};
+
+		if (this.isEnabled) {
+			await this.sendTransactionalMail({
+				template: emailInfo.template,
+				data,
+			});
+		} else {
+			this.logger.log(
+				`Mock email sent. To: '${data.to}'. Template: ${
+					emailInfo?.template
+				}, data: ${JSON.stringify(data)}`
 			);
 			return false;
 		}
+		return true;
+	}
 
-		const cmTemplateId = templateIds[emailInfo.template];
-		if (!cmTemplateId) {
-			this.logger.error(
-				`Campaign monitor template ID for ${emailInfo.template} not found -- email could not be sent`
-			);
-			return false;
+	// TODO: Write tests (ARC-1500)
+	public async sendForMaterialRequest(emailInfo: MaterialRequestEmailInfo): Promise<boolean> {
+		const recipients: string[] = [];
+
+		if (emailInfo.to) {
+			recipients.push(emailInfo.to);
+		} else {
+			if (emailInfo.isToMaintainer) {
+				this.logger.error(
+					`Mail will not be sent to maintainer id ${emailInfo.materialRequests[0]?.maintainerId} - empty email address`
+				);
+			} else {
+				this.logger.error(
+					`Mail will not be sent to profile id ${emailInfo.materialRequests[0]?.profileId} - empty email address`
+				);
+			}
 		}
 
 		const data: CampaignMonitorData = {
 			to: recipients,
 			consentToTrack: 'unchanged',
-			data: this.convertVisitToEmailTemplateData(emailInfo.visit),
+			data: this.convertMaterialRequestsToEmailTemplateData(
+				emailInfo,
+				emailInfo.firstName,
+				emailInfo.lastName
+			),
 		};
 
 		if (this.isEnabled) {
-			await this.sendMail({
-				templateId: cmTemplateId,
+			await this.sendTransactionalMail({
+				template: emailInfo.template,
 				data,
 			});
 		} else {
 			this.logger.log(
-				`Mock email sent. To: '${
-					data.to
-				}'. Template: ${cmTemplateId}, data: ${JSON.stringify(data)}`
+				`Mock email sent. To: '${data.to}'. Template: ${
+					emailInfo?.template
+				}, data: ${JSON.stringify(data)}`
 			);
 			return false;
 		}
```

Comment:
This check for if isEnabled can be moved inside the sendTransactionalMail function, so it isn't duplicated for sendForVisit and sendForMaterialRequest